docs/metal_device_create.md

```diff
@@ -39,7 +39,7 @@ metal device create -p <project_id> (-m <metro> | -f <facility>) -P <plan> -H <h
   -s, --spot-instance                    Provisions the device as a spot instance.
       --spot-price-max float             Sets the maximum spot market price for the device: --spot-price-max=1.2
   -t, --tags strings                     Tag or list of tags for the device: --tags="tag1,tag2".
-  -T, --termination-time string          Device termination time: --termination-time="15:04:05"
+  -T, --termination-time string          Device termination time: --termination-time="2023-08-24T15:04:05Z"
```

Comment:
IMO this is a positive change (the old example is ambiguous), but I just want to know if the old example (`15:04:04` with no date or TZ indicator) actually worked, and if so, what termination time did the device end up with?

Reply:
Old example is not working with metal-go